### PR TITLE
Fixed #212 UX issues and question/response wording.

### LINKED ISSUE #212: Minor adjustments to new "first TPS?" question
This new question, id: type of application, works perfectly. I have some suggestions for UX.

1. Can the new question be moved to right after id: signpost for main application questions? I think swapping the order of these two lines of code would do it. 
```
      user.is_first_TPS_application
      user.signpost
```
2. Can we adjust the question text so it reads: First TPS application for ${ users[i] }? 
3. Can we change the label for the first/re-registering field to align it with other questions (in 2nd person): Is this your first TPS application, or are you re-registering/reapplying for TPS?
4. Can we change the 2nd label of TPS-grantor to: Who previously granted your TPS?

### PR DESCRIPTION
**Pull request by ll-jgraszer**
Fix #212

### CODE REVIEW

**Review by mnewsted — CHANGES_REQUESTED** [review]
Thanks @ll-jgraszer ! This looks good.

I realized I oversimplified a bit of the language. What do you think of changing line 644 from "...are you re-registering/reapplying for TPS" to "...are you applying to re-register your TPS?" That is slightly more in line with the application. I think the choices are fine.

After that tweak, I think this can be merged. I'll then test it again before deployment. Thank you!

**Comment by ll-jgraszer** [comment]
@mnewsted Fixed the language pending your approval. Thanks!

**Review by mnewsted — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**
